examples/basic/legendSpecificAttrs_option.html

```diff
@@ -1592,4 +1592,4 @@
 
 
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
I tried to commit without newline, but it doesn't seem to work. I don't know why >_<